pavelib/quality.py

```diff
@@ -298,6 +298,50 @@ def run_jshint(options):
         )
 
 
+@task
+@needs('pavelib.prereqs.install_python_prereqs')
+@cmdopts([
+    ("limit=", "l", "limit for number of acceptable violations"),
+])
+def run_safelint(options):
+    """
+    Runs safe_template_linter.py on the codebase
+    """
+
+    violations_limit = int(getattr(options, 'limit', -1))
+
+    safelint_report_dir = (Env.REPORT_DIR / "safelint")
+    safelint_report = safelint_report_dir / "safelint.report"
+    _prepare_report_dir(safelint_report_dir)
+
+    sh(
+        "/edx/app/edxapp/edx-platform/scripts/safe_template_linter.py >> {safelint_report}".format(
```

Comment:
@robrap I'd do something like [this](https://github.com/edx/edx-platform/blob/b8dc2af3a794c7b8875d24279e11625836ac46bd/pavelib/utils/test/suites/bokchoy_suite.py#L152)